packages/playground-bundling/scripts/generate_cmijs.js

```diff
@@ -75,7 +75,7 @@ function buildCompilerCmij() {
   }
 
   e(
-    `find ${rescriptLibOcamlFolder} -name "*.cmi" -or -name "*.cmj" | xargs -n1 basename | xargs js_of_ocaml build-fs -o ${cmijFile} -I ${rescriptLibOcamlFolder}`,
+    `find ${rescriptLibOcamlFolder} -name "*.cmi" -or -name "*.cmj" | xargs -n1 basename | xargs js_of_ocaml build-fs -o ${cmijFile} -I ${rescriptLibOcamlFolder}`
```

Comment:
I really thought this line would need to be changed, but I am glad this is just formatting.

Reply:
@fhammerschmidt would you like the formatting noise to be removed?